libs/components/src/next/properties/vl-properties.component.ts

```diff
@@ -0,0 +1,133 @@
+        return this.renderDl();
+    }
+
+    protected firstUpdated(_changedProperties: PropertyValues) {
+        super.firstUpdated(_changedProperties);
+        if (this.shadowRoot) {
+            this.shadowRoot.adoptedStyleSheets = [...this.shadowRoot.adoptedStyleSheets, this.labelWidthSheet];
+        }
+    }
+
+    updated(changedProperties: Map<string, unknown>) {
+        super.updated(changedProperties);
+
+        if (changedProperties.has('labelWidth') && this.labelWidth) {
+            this.labelWidthSheet.replace(labelWidthRem(this.labelWidth).toString());
+        }
+    }
+
+    private renderDl(): TemplateResult {
+        return html` <dl>${this.aggregatedProps.map((column: Column) => this.renderColumn(column))}</dl> `;
+    }
+
+    private renderColumn(column: Column): TemplateResult {
+        return column.class
+            ? html` <div class="${column.class}">${column.items.map((item: Item) => this.renderItem(item))}</div>`
+            : html` ${column.items.map((item: Item) => this.renderItem(item))}`;
+    }
+
+    private renderItem(item: Item): TemplateResult {
+        return html` <div class="item">${this.renderLabels(item.labels)} ${this.renderData(item.data)}</div>`;
+    }
+
+    private renderLabels(labels: string[] | Node[][]): TemplateResult[] {
+        return labels.map((label: string | Node[]) => html` <dt>${label}</dt>`);
+    }
+
+    private renderData(itemData: string[] | Node[][]): TemplateResult[] {
+        return itemData.map((data: string | Node[]) => html` <dd>${data}</dd>`);
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+        this.buildInternalProperties();
+        this.observeLitePropertiesChange();
+    }
+
+    private buildInternalProperties() {
+        this.aggregatedProps = [...this.attributeProps, ...buildProperties([...this.children])];
+    }
+
+    private observeLitePropertiesChange() {
+        // verwijder alle bestaande mutation observers
+        this.mutationObserverList.forEach((mutationObserver) => mutationObserver.disconnect());
+        this.mutationObserverList = [];
+        // altijd de vl-properties zelf observeren
+        this.mutationObserverList = [
+            ...this.mutationObserverList,
+            onChildListChange(this, (change: any) => {
+                this.buildInternalProperties();
+                this.requestUpdate();
+                // er kan een extra div toegevoegd zijn, deze heeft dan ook een mutation observer nodig
+                this.observeLitePropertiesChange();
+            }),
+        ];
+        // als de directe kinderen div's zijn deze ook observeren
+        if (this.children.length > 0 && this.children[0].localName === 'div') {
+            [...this.children].forEach(
+                (element) =>
+                    (this.mutationObserverList = [
+                        ...this.mutationObserverList,
+                        onChildListChange(element, (change: any) => {
```

Comment:
Kan je de MutationObserver die hier aangemaakt wordt bijhouden en disconnecten in een disconnectedCallback?